training/src/main/scala/com/airbnb/aerosolve/training/pipeline/GenericPipeline.scala

```diff
@@ -156,6 +156,7 @@ object GenericPipeline {
     val isRegression = Try(cfg.getBoolean("is_regression")).getOrElse(false)
     val isMulticlass = Try(cfg.getBoolean("is_multiclass")).getOrElse(false)
     val metric = cfg.getString("metric_to_maximize")
+    val resultsOutputPath = Try(cfg.getString("results_output_path")).getOrElse("")
```

Comment:
seems not yet push? i.e. demo https://github.com/airbnb/aerosolve/tree/master/demo/income_prediction